src/blocks.js

```diff
@@ -32,6 +49,48 @@ export default (editor, opts = {}) => {
     });
   });
 
+  /*
+   * Blocks for Preference Center Category
+   */
+  //check if page is for preference center
+  if (mQuery('#page_isPreferenceCenter_1').is(':checked')) {
+    if (typeof bm.get('MyCategories') !== 'undefined') {
```

Comment:
This code is only used with the preferenceCenter. I think it makes sense to move it to the place where the rest of the preference center code is: preferenceCenter/